stdlib/build.rs

```diff
@@ -57,12 +57,24 @@ trait Renderer {
 }
 
 /// Writes Miden standard library modules documentation markdown files based on the available modules and comments.
-pub fn build_stdlib_docs(module_map: &ModuleMap, output_dir: &str) {
-    // Remove functions folder to re-generate
-    fs::remove_dir_all(output_dir).unwrap();
-    fs::create_dir(output_dir).unwrap();
+pub fn build_stdlib_docs(module_map: &ModuleMap, output_dir: &str) -> io::Result<()> {
```

Comment:
Does this function actually get run when running `cargo doc`? I thought that line 24 above should prevent this - but maybe it doesn't work this way?

Reply:
Here is where I looked:

https://docs.rs/crate/miden-stdlib/latest

![image](https://github.com/0xPolygonMiden/miden-vm/assets/310139/925e2864-309f-42b4-9eaf-5c17655c2227)

Reply:
This seems to be the documented approach https://docs.rs/about/builds#detecting-docsrs